src/resolvers/archDockerfile.ts

```diff
@@ -10,7 +12,7 @@ import { Bundle, FileInfo, Resolver } from '../resolver';
 type ArchSpecificDockerfile = [string, FileInfo];
 
 export default class ArchDockerfileResolver implements Resolver {
-	public priority = 1;
+	public priority = 3;
```

Comment:
I admit that the story for changing priority isn't great, but it is possible, albeit clunky. By defining their own resolvers, and also using the default ones, a user of this module could change the priority value themselves. 

I do think it needs more thought, but at the moment the only user of this module is resin-cli and the builder, both of which are happy with the default resolvers and priorities. I'm happy to chat over alternative methods of doing this with you, and adding an issue to be tackled when time allows.